src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/NativeTypes.cs

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace System.DirectoryServices.Protocols
+{
+    internal static class AuthConstants
```

Comment:
The right place for this might be src/libraries/Common/src/Interop/Interop.ldap.cs, wrapped in `internal partial class Interop { internal partial class Ldap { ... } }` (and no namespace).

Reply:
(But only for the parts that are truly shared across Windows and Linux.  Anything platform-specific would go in an appropriate subdirectory of Interop)